Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.24 AS builder
+ARG CGO_ENABLED=0
+ARG APP_NAME
+WORKDIR /app
+
+COPY go.mod go.sum ./
+RUN go mod download
+COPY . .
+
+RUN go build -o /app/$APP_NAME ./cmd/$APP_NAME
+
+FROM ghcr.io/games-on-whales/base-app:edge AS output
+ARG APP_NAME
+
+WORKDIR /app
+
+COPY images/$APP_NAME/startup.sh /app/entrypoint.sh
```

Comment:
Probably want to copy it to `/opt/gow/startup.sh` instead, and dont override `ENTRYPOINT` as that is already `/entrypoint.sh` which does some pre startup things before launching /opt/gow/startup.sh